### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by ragangithub**
- Created app.rb file that will serve as console app entry-point. It has the following h methods 
    List all books.
    List all people.
   Create a person (teacher or student, not a plain Person).
   Create a book.
   Create a rental.
    List all rentals for a given person id.
- Created main.rb define the entry point,  This  file has a method that  does the following:
    Present the user with a list of options to perform.
    Lets users choose an option.
    If needed, ask for parameters for the option.
    quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    when 2
+      print 'What is the teachers specialization: '
+      specialization = gets.chomp
+
+      teacher = Teacher.new(name, age, specialization, parent_permission: true)
+
+      @person << teacher
+    end
+
+    print 'Success'
```
Comment (KanzaTahreem) [inline app.rb:58]: - [ ] Good job for creating a teacher ✨ However, the teacher's attributes are misplaced on listing all people. Please make sure to fix this ⛏

Current UI | Expected UI
---| ---|
image | image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @ragangithub 

Good job so far 👏! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good job with the UI
- ✅ Descriptive PR and README file
- ✅ Correct Linters

## Required Changes 🛑

_Please check the comments under the review._

## Optional suggestions ❕

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ragangithub** [comment]
- corrected teacher's attributes

**Review by Munsa1 — APPROVED** [review]
Hi @ragangithub, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Fixed Requested Change By Previous Reviewer, Teacher Data Is As Entered :heavy_check_mark: 
image



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**